src/MediaWiki.ts

```diff
@@ -296,6 +401,12 @@ class MediaWiki {
 
     return mwMetaData
   }
+
```

Comment:
A few comments have been lost here, maybe on purpose?! Just try to keep things commented when needed.